### PR TITLE
Underscored usernames

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This fixes #642 the way @gvwilson wanted.

### CODE REVIEW

**Comment by gvwilson** [comment]
Thanks @pbanaszkiewicz

**Comment by gvwilson** [comment]
See also https://github.com/swcarpentry/certification/pull/9

**Comment by pbanaszkiewicz** [comment]
@gvwilson okay, you have a better idea what has to be done yet to migrate everything to underscores. Can you please point me where I could help?

**Comment by gvwilson** [comment]
I'll work on this one next week.

File: workshops/util.py [hunk 1]
```diff
@@ -297,27 +297,25 @@ def create_uploaded_persons_tasks(data):
     return persons_created, tasks_created
 
 
-def create_username(personal, family):
+def create_username(personal, family, tries=100):
```
Comment (gvwilson) [inline workshops/util.py]: Put the 100 in a constant at the top of the file so it's findable.

File: workshops/util.py [hunk 2]
```diff
@@ -326,6 +324,9 @@ def normalize_name(name):
     for (accented, flat) in [(' ', '-')]:
         name = name.replace(accented, flat)
 
+    # remove all non-ASCII, non-hyphen chars
```
Comment (gvwilson) [inline workshops/util.py]: the comment is incorrect: `.` and `'` are ASCII characters, but this removes them.

**Pull request merged**